elintegrostartapp/grails-app/conf/spring/globech/resourcesScriptBean.groovy

```diff
@@ -764,10 +764,12 @@ beans {
                                                                    var response = responseData.data;
                                                                    self.vueCreateProjectForTranslationDataframe_project_sourceFile_ajaxFileSave(response,allParams);
                                                                    excon.saveToStore('vueTranslatorAssistantDataframe','vueTranslatorAssistantDataframe_project_list',response.params.name);
-                                                                   excon.saveToStore('vueTranslatorAssistantDataframe','currentProjectId',response.params.id)
-                                                                   excon.setVisibility('vueCreateProjectForTranslationDataframe',false);
+                                                                   excon.saveToStore('vueTranslatorAssistantDataframe','currentProjectId',response.params.id);
+                                                                   setTimeout(function(){excon.setVisibility('vueCreateProjectForTranslationDataframe', false);}, 4000);
```

Comment:
Pleas open a ticket that asks to find a way to call setTimeout for every Dataframe by default and option to override it here in resourceScriptBean.groovy